source/regions/flags.rst

```diff
@@ -293,7 +293,7 @@ Map Making
     firework-damage,state,Whether entities receive damage from fireworks
     game-mode,gamemode,"The gamemode (survival, creative, adventure) that will be applied to players that enter the region"
     time-lock,string,"Time of day in ticks (between 0 and 24000) that players will see the world as while in the region. Use + or - for time relative to the world time."
-    weather-lock,weather,Type of weather players will see when in the region. This does not affect world mechanics. Valid values are ``downfall`` and ``clear``.
+    weather-lock,weather,Type of weather players will see when in the region. This does not affect world mechanics. Valid values are ``rain`` and ``clear``.
```

Comment:
```suggestion
    weather-lock,weather,Type of weather players will see when in the region. This does not affect world mechanics. Valid values are ``rain``, ``thunder_storm``, and ``clear``.
```

Reply:
You know while this valid it just occurred to me that the bukkit API doesn't support this, so it's probably not worth mentioned it.